src/Microsoft.Crank.Agent/Startup.cs

```diff
@@ -3704,6 +3704,16 @@ private static async Task PatchProjectFrameworkReferenceAsync(Job job, string pr
                         );
                     }
 
+                    // Exclude "published" folder from content to prevent recursively copying it after each cached build
+                    var excludePublishedGroup = new XElement("ItemGroup");
+                    project.Root.Add(excludePublishedGroup);
```

Comment:
Should this be added after we update the `excludePublishedGroup`? Just in case it copies on add.

Reply:
You might be right, I see something like this:

```xml
  <ItemGroup />
  <ItemGroup>
    <Content Update="published\**" CopyToPublishDirectory="Never" />
  </ItemGroup>
```